src/netspresso_trainer/models/backbones/experimental/darknet.py

```diff
@@ -2,18 +2,36 @@
 Based on the Darknet implementation of Megvii.
 https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/models/darknet.py
 """
-from typing import Dict, Optional, List
+
+from typing import Dict, Optional, List, Type
 
 from omegaconf import DictConfig
 import torch
 from torch import nn
 
-from ...op.custom import ConvLayer, CSPLayer, Focus, SPPBottleneck
+from ...op.custom import (
+    ConvLayer,
+    CSPLayer,
+    Focus,
+    SPPBottleneck,
+    Bottleneck,
+    BasicBlock,
+    DarknetBlock,
+)
 from ...utils import BackboneOutput
 from ..registry import USE_INTERMEDIATE_FEATURES_TASK_LIST
 
-__all__ = ['cspdarknet']
-SUPPORTING_TASK = ['classification', 'segmentation', 'detection', 'pose_estimation']
+__all__ = ["cspdarknet"]
+SUPPORTING_TASK = ["classification", "segmentation", "detection", "pose_estimation"]
+
+
+BLOCK_FROM_LITERAL: Dict[str, Type[nn.Module]] = {
+    "basicblock": BasicBlock,
+    "bottleneck": Bottleneck,
+    "darknetblock": DarknetBlock,
+}
```

Comment:
I think `BLOCK_FROM_LITERAL` is redundant.
Repeating blocks in YOLO-Fastest is actually `depthwise` version of `DarknetBlock`. If you activate `depthwise` parameter in `DarknetBlock`, This and related lines can be removed.

https://github.com/Nota-NetsPresso/netspresso-trainer/blob/48217e8901efd1cb4ee9851d5bfcafc93ef9845e/src/netspresso_trainer/models/op/custom.py#L608-L631

About `DarknetBlock` with `depthwise`, refer here.

https://github.com/Megvii-BaseDetection/YOLOX/blob/ac58e0a5e68e57454b7b9ac822aced493b553c53/yolox/models/network_blocks.py#L79-L101